source/connection_manager.c

```diff
@@ -265,6 +292,7 @@ struct aws_http_connection_manager_snapshot {
     size_t idle_connection_count;
     size_t pending_acquisition_count;
     size_t pending_connects_count;
+    size_t pending_settings_count;
     size_t vended_connection_count;
     size_t open_connection_count;
```

Comment:
if any of these are the the sum of others please comment

really they could all use a nice 5 word description

Reply:
Hmmm, I realize that they all have detailed description from the connection manger structure. Like [here](https://github.com/awslabs/aws-c-http/pull/360/files#diff-60eb9519b6fcfa87a65c15d68252caf78293eaff955953d4da882c3876186354R204). So, maybe we should be good here.